### PR TITLE
Laravel 5 support for Tiny-PHP

### PR DESCRIPTION
**Pull request by aranw**
I've added the additional folder `Laravel5` to provide non breaking changes to the Laravel 4 Service Provider and command. 

Not sure if you'd like to tackle this another way though?

### CODE REVIEW

**Comment by mikelbring** [comment]
+1

**Comment by zackkitzmiller** [comment]
@mikelbring You feel good about this, we can merge.

File: src/ZackKitzmiller/Laravel5/TinyGenerateCommand.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        if (file_exists($path) and getenv('LEAGUE_TINY_KEY') !== false) {
+            $originalContent = file_get_contents($path);
+            $content = str_replace('LEAGUE_TINY_KEY='.getenv('LEAGUE_TINY_KEY'), 'LEAGUE_TINY_KEY='.$key, $originalContent);
+
+            file_put_contents($path, $content);
+        } else {
+            $fp = fopen($path, 'a');
+            fwrite($fp, "\nLEAGUE_TINY_KEY=$key\n");
+            fclose($fp);
+        }
```
Comment (vlakoff) [inline src/ZackKitzmiller/Laravel5/TinyGenerateCommand.php]: How about: `file_put_contents($path, "\nLEAGUE_TINY_KEY=$key\n", FILE_APPEND);`

**Comment by amirhs712** [comment]
@zackkitzmiller using version 1.3.0 on Laravel 5.8, PHP 7.2
Service Provider:  ZackKitzmiller\Laravel5\TinyServiceProvider::class

The package is not discovered by Laravel, and when I'm trying to use php artisan tiny:xxx commands
I get the following:

`ReflectionException  : Method ZackKitzmiller\Laravel5\TinyGenerateCommand::handle() does not exist
`

**Update:**
Changing the fire function to handle, inside of the `TinyGenerateCommand` fixes the problem.
However, changing the source code is not a good idea.

**Pull request merged**